include/sound/sof/topology.h

```diff
@@ -33,12 +33,13 @@ enum sof_comp_type {
 	SOF_COMP_EQ_IIR,
 	SOF_COMP_EQ_FIR,
 	SOF_COMP_KEYWORD_DETECT,
-	SOF_COMP_KPB,			/* A key phrase buffer component */
-	SOF_COMP_SELECTOR,		/**< channel selector component */
+	SOF_COMP_KPB,		/**< key phrase buffer component */
+	SOF_COMP_SELECTOR,	/**< channel selector component */
 	SOF_COMP_DEMUX,
-	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
+	SOF_COMP_ASRC,		/**< asynchronous sample rate converter */
```

Comment:
@kv2019i well, they are semi-related, but yes, I can make a separate patch for them

Reply:
Ack @lyakh Otherwise it's fix this kind of stuff when doing changes closeby, but considering your series is more complex than usual, you need to touch some core bits of SOF and that is a long-running series (as it is doing new stufff) , I'd minimize all code that can add to merge conflicts.